proctor_exam/proctor_exam.py

```diff
@@ -118,13 +120,19 @@ def get_proctorexam_user_state(self, exam_id, lti_parameters):
         signature = hmac.new(secret_key.encode(), fields.encode(), hashlib.sha256).hexdigest()
         params["signature"] = signature
 
+        user_not_setup = {"student": {"status": "not_setup"}}
         response = requests.get(endpoint_url, headers=headers, data=json.dumps(params))
+
+        if not response.ok:
```

Comment:
Shouldn't you be more specific to match only 404 errors ?
To avoid returning `user_not_setup` if the remote server return a 500 due to an outage for example.

Reply:
true that. In this case I could display an error and ask to call support or something like that :sweat_smile: